tee-worker/litentry/pallets/identity-management/src/migrations.rs

```diff
@@ -61,3 +63,16 @@ pub fn migrate_to_v2<T: crate::Config, P: GetStorageVersion + PalletInfoAccess>(
 	}
 	Weight::zero()
 }
+
+pub fn drop_web3networks_from_id_graph<T: crate::Config>() -> Result<Weight, Error<T>> {
```

Comment:
Is this function called only in tests? How is migration performed ?

Reply:
yeah, I'm on the way to figuring out where to put it to trigger on runtime upgrade. And according to our guide https://www.notion.so/web3builders/How-to-upgrade-enclave-one-worker-edfaf5871b4441579f9471074032ed1e

the best place is `tee-worker/app-libs/sgx-runtime/src/migration.rs`